src/structured-data/structured-data-tags.js

```diff
@@ -218,6 +218,14 @@ function generateBreadcrumbListData(pageType = "", publisherConfig = {}, data =
     return sectionCrumbsDataList;
   }
 
+  function getCustomPageCrumbs(collection = {}, breadcrumb = {}) {
```

Comment:
make this `getBreadCrumbs`. You dont need to pass collection to it. Just accept breadcrumb and pass only the list to `addCrumb`

Reply:
There you go. I have changed **getCustomPageCrumbs** to **getBreadCrumbs** and removed **collection** argument. Can you please check this once?

Reply:
@sharangj Any update on this?